src/js/selection.js

```diff
@@ -375,12 +374,11 @@
                 sel = doc.getSelection();
 
             range.selectNodeContents(node);
-            sel.removeAllRanges();
-            sel.addRange(range);
+            MediumEditor.util.replaceRange(sel, range);
         },
 
         select: function (doc, startNode, startOffset, endNode, endOffset) {
-            doc.getSelection().removeAllRanges();
+            MediumEditor.util.removeAllRanges(doc.getSelection());
```

Comment:
It's safe to get rid of this line and just call `MediumEditor.util.removeAllRanges()` on line 389.  That would allow us to not have 2 new helpers exposed (the reason why I like to try to avoid adding to many exposed helpers is that we can't really change the signature or the behavior of those without doing a major release, so the less we expose externally the better).